app/src/main/java/com/kickstarter/ui/views/CommentReplyCard.kt

```diff
@@ -0,0 +1,99 @@
+package com.kickstarter.ui.views
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import androidx.core.content.withStyledAttributes
+import androidx.core.view.isVisible
+import com.kickstarter.R
+import com.kickstarter.databinding.CommentReplyCardBinding
+import com.kickstarter.ui.extensions.loadCircleImage
+
+class CommentReplyCard @JvmOverloads constructor(
```

Comment:
Do we really need to create another Component for the reply? it seems a lot of duplicated code just in order to add a margin on the left. 
We could re-use the already existing `CommentCard` and use these states or add a new one:
<img width="512" alt="Screen Shot 2021-06-01 at 12 09 39 PM" src="https://user-images.githubusercontent.com/4083656/120377659-461a9500-c2d2-11eb-9ce9-4e1dfbb62d08.png">

  to add a (guideline and left constraint to it in case is a reply) or simply a marginleft, not really strong opinions in how to build that part of the UI

Reply:
thought about that. There are several logics around the card state. Did not want to complicate it. I will try reusing the existing commentCard.

Reply:
For sure that simpler than adding the margin on the card UI component, great suggestion 👍